document_page_approval_levels/models/document_page_item.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 206 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: Luis Torres <luis_t@vauxoo.com>
+#    planned by: Sabrina Romero <sabrina@vauxoo.com>
+############################################################################
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning as UserError, ValidationError
+
+
+class ApproverDocumentPageItem(models.Model):
+    _name = 'approver.document.page.item'
+
+    user_id = fields.Many2one('res.users', 'User')
+
+    approved = fields.Boolean()
+
+    document_id = fields.Many2one('document.page', 'Document')
+
+    history_id = fields.Many2one('document.page.history', 'History')
+
+    @api.multi
+    def approve(self):
+        """Approve a document page and send an note in the register."""
+        for item in self:
+            if item.approved:
+                continue
+            if item._uid != item.user_id.id:
+                raise UserError(
+                    _('Invalid Action!'),
+                    _('Only the user assigned to this item can validate it.'))
+            item.write({'approved': True})
+            msg = (
+                '<div><b>The page has been approved by %s .</b></div>' %
+                item.user_id.name)
+            self.env['mail.message'].create({
```

Comment:
Use message_post not .create directly.

Do an rgrep into sale and purchase module to understand the usage.